app/assets/javascripts/components/story/ExpandedStory/ExpandedStoryDescription.js

```diff
@@ -35,22 +37,30 @@ class ExpandedStoryDescription extends React.Component {
     );
   };
 
-  descriptionTextArea(description) {
-    const { onEdit, disabled } = this.props;
+  descriptionTextArea(description, suggestedUsers) {
+    const { disabled, onEdit } = this.props;
 
     return (
-      <textarea
+      <MentionsInput
         className="form-control input-sm edit-description-text"
         onChange={(event) => onEdit(event.target.value)}
         readOnly={disabled}
         value={description}
-      />
+      >
+        <Mention
+          markup="@__display__"
+          displayTransform={(id, display) => `@${display}`}
```

Comment:
What do you think about extracting this to a class function to avoid recreating it in every re-render?

Reply:
You mean, using a method to return this value?? :thinking:

Reply:
Yes, just like you did ;)